### PR TITLE
Custom help content for "Wrong WordPress.com account error" screen

### PR DESCRIPTION
**Pull request by selanthiraiyan**
Part of: #7547 

### Description
As a part of improving the help center content, this PR adds custom content with FAQs and Answers for the "Wrong WordPress.com account error" screen at the following URL.

https://woocommerce.com/document/android-ios-apps-login-help-faq/#wrong-account

Internal ref - pe5sF9-sZ-p2

### Testing instructions

1. Install and launch the app.
1. Tap `Skip` to reach the login prologue screen.
1. Tap the `Enter Your Store Address` button and enter store address.
1. Tap `Continue`, and you will land in the "Enter WordPress.com account email" screen.
1. Sign in using a WordPress.com account email which isn't connected with the store's Jetpack.
1. You will land in the following wrong account error screen. 

7. Tap `Help` -> `Help Center`
1. Observe that you are directed to https://woocommerce.com/document/android-ios-apps-login-help-faq/#wrong-account in a web view
1. In Xcode debug console observe that the following event is tracked. Note that the `source_flow` value will differ based on the flow you selected to login.
```
Tracked support_help_center_viewed, properties: [AnyHashable("source_flow"): "login_password_magic_link_emphasis", AnyHashable("help_content_url"): "https://woocommerce.com/document/android-ios-apps-login-help-faq/#wrong-account", AnyHashable("source_step"): "wrong_wordpress_account"]
```

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by itsmeichigo — APPROVED** [review]
Everything looks good and works as expected ✅ 

I have a non-blocking question: Is the web page content something we can take care of? From what I see, we could explain it a bit further to make it more helpful to users. 

E.g, "the store you are trying to login into is connected to a different WordPress.com account" sounds like the store can only be connected to one WP.com account. We can reword this to something like "the WordPress.com account you are trying to log in to is not connected to your site's Jetpack". 

We can also add a new section to point them to how to handle the connection, like what we're doing as part of [pe5sF9-AL-p2](pe5sF9-AL-p2).

**Comment by selanthiraiyan** [comment]
Thanks for the review and inputs @itsmeichigo 🙇 

> I have a non-blocking question: Is the web page content something we can take care of? From what I see, we could explain it a bit further to make it more helpful to users.

I have created [a new subtask](https://github.com/woocommerce/woocommerce-ios/issues/7547) to work on fixing and adding more details to this section of the help center page.
